output cn, ct, cl, cd for all cases.

juddmehr opened the pull request:
Not sure if this is breaking or not.  But I added some more outputs when the VPM is running as well (the previous additions were just for the quazi-steady solver)

EdoAlvarezR reviewed (changes requested):
Line 1143 overwrite `turbine_flag` to be always true. Are you sure of that change? This seems dangerous. 

Also, please define a send me a test case of a rotor in propeller mode showing that you have verified that nothing broke or change in between before and after these changes.

juddmehr commented:
[temp_singlepropeller_master.zip](https://github.com/byuflowlab/FLOWVLM/files/5240012/temp_singlepropeller_master.zip)
[temp_singlepropeller_pullreq.zip](https://github.com/byuflowlab/FLOWVLM/files/5240014/temp_singlepropeller_pullreq.zip)

Here are the outputs before and after the changes.  I also updated the single prop example to work in Julia 1.x.

I fixed the hardcoded turbine flag as well.
